src/amo/components/ScreenShots/index.js

```diff
@@ -79,6 +79,7 @@ export default class ScreenShots extends React.Component<Props> {
                       src={preview.thumbnail_src}
                       width={preview.thumbnail_w}
                       height={preview.thumbnail_h}
+                      loading="lazy"
```

Comment:
Adding an assertion for this would be good.

Reply:
like, an `expect` line in the tests?  Or something else?

Reply:
yes an `expect` to prevent the future AMO team to remove this prop because it's useless :p

Reply:
https://github.com/mozilla/addons-frontend/pull/10189/commits/eafe9ecfd3574b68b647187553d8d05bc9082ff0